src/verus.pest

```diff
@@ -1007,7 +1019,7 @@ verusfmt_skipped_expr_no_struct = {
 
 // This split of `expr` and `expr_inner` is to simply break the left-recursion
 // that would happen otherwise.
-expr = {
+expr = !{
```

Comment:
I'm surprised this became atomic and that doing so didn't break things.  Can you provide some intuition on both fronts?

Reply:
Same as the other case, this is explicitly marking it as non-atomic (even in an atomic context).